client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+import './style.scss';
+import { useTranslate } from 'i18n-calypso';
+
+const PluginNavbarHeader = () => {
+	const translate = useTranslate();
+
+	return (
+		<div>
+			<div className="x-root lpc-header-nav-wrapper">
+				<div className="lpc-header-nav-container">
+					{ /*<!-- Nav bar starts here. -->*/ }
+					<div className="masterbar-menu">
+						<div className="masterbar">
+							<nav className="x-nav" aria-label={ translate( ' WordPress.com' ) }>
+								<ul className="x-nav-list x-nav-list--left">
+									<li className="x-nav-item">
+										<a
+											role="menuitem"
+											className="x-nav-link x-nav-link--logo x-link"
+											href="//wordpress.com/"
```

Comment:
Should this lead to `/` instead, so we would keep the current environment if we're not in staging/production?